pkg/config/config.go

```diff
@@ -620,6 +620,7 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("snmp_traps_enabled", false)
 	config.BindEnvAndSetDefault("snmp_traps_config.port", 162)
 	config.BindEnvAndSetDefault("snmp_traps_config.community_strings", []string{})
+	config.BindEnvAndSetDefault("snmp_traps_config.namespace", "default")
```

Comment:
```suggestion
	config.SetKnown("snmp_traps_config.namespace")
```

Do we need a default value here since we already rely on `network_devices.namespace` as fallback in ReadConfig() ?